roles/install_storpool_components/handlers/main.yml

```diff
@@ -29,6 +29,7 @@
   ansible.builtin.reboot:
     reboot_timeout: 900
   listen: Reboot host
+  throttle: "{{ 1 if sp_reboot_sequentially else 63 }}"
```

Comment:
Better use the `ternary` filter

Reply:
I thought this was python's ternary operator? If you mean something else - I need more explanations :)

Reply:
I'm referring to this: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/ternary_filter.html